Test Linters of HTML and CSS

zbahati opened the pull request:
## This Pull request is created to test the Linters 
we install : 
- Webhint 
- stylelint.json
- hintrc
- lighthouse

Commit:
Add Files and Folders
 - Add .github Folder
 - Add .gitignore File
 - Add .hintrc File
 - Add .stylelint.json

Commit:
Add index.html file
- Add header in it
- Add styling of header

Commit:
Add style.css File
- Add class called title on header
- Add styling of Class and give color

Commit:
Fix Lighthouse error

AlphaNtihinduka commented on README.md:
```diff
@@ -1 +1,288 @@
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [your_project_name] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[your_project__name]** is a...
+
```
- [ ] Please consider adding a descriptive project name, this will make the Readme professional and give an overview to the users about the project ☺️. You need to consider removing the default placeholder ` Describe your project in 1 or 2 sentences`

AlphaNtihinduka commented on README.md:
```diff
@@ -1 +1,288 @@
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```
- [ ] I have noticed the image is not displayed, kindly ensure that the image on the readme can be seen by the users.

AlphaNtihinduka commented on README.md:
```diff
@@ -1 +1,288 @@
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[key_feature_1]**
+- **[key_feature_2]**
+- **[key_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
- [ ] Please try as much as possible to customize the README template. For this section, we need to ensure that we provide the `key features` of the `Hello Microverse` project. In addition, as previously requested, you may consider removing the place holders in this section. Finally, I have noticed that the `live Demo Link` is broken. Kindly ensure that you attached deployed site and if not yet deployed you may consider stating that ☺️.

AlphaNtihinduka commented on README.md:
````diff
@@ -1 +1,288 @@
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+
+Install this project with:
+
+<!--
+Example command:
````
- [ ] Kindly consider customizing the `setup`, `install`, `usage`, `run tests`, and `deployment` subsections. If for this project one of the subsection is not applied you can consider stating this. For example `install` since there are no gems or dependencies to install 😞.

AlphaNtihinduka commented on README.md:
```diff
@@ -1 +1,288 @@
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
- [ ] Again, we are on the same part of customization, consider using your social handles, so that when a user clicked on them, he is directed to your social media or GitHub page ☺️. This will help us to make our README file professional. Additional consider removing the default placeholders as stated earlier.

AlphaNtihinduka commented on README.md:
```diff
@@ -1 +1,288 @@
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
```
- [ ] Kindly ensure to customize these sections also and remove the placeholders. This is to ensure that the README file is professional ☺️.

AlphaNtihinduka commented on README.md:
```diff
@@ -1 +1,288 @@
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
- [ ] [OPTIONAL] However much this is optional section, but to have a professional file, we can consider customizing it to ensure that recruiters are pleased with your project.

AlphaNtihinduka commented on README.md:
```diff
@@ -1 +1,288 @@
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
- [ ] [OPTIONAL] Kindly ensure that you add the `MIT` by creating a license file in the project. The MIT license gives users :person_curly_hair:  express permission to reuse code for any purpose.

AlphaNtihinduka reviewed (changes requested):
## Hi @zbahati 🙋‍♂️
Good job working on your Hello Microverse 👏.

You still need to implement few on the project but you are almost there!!! :watch: 

TO HIGHLIGHT:
- Your PR is professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

![close](https://user-images.githubusercontent.com/99394374/210771148-619a4265-33b2-4e68-8901-6bb52693d6de.gif)

# Required Changes :recycle: 
- _Kindly consider the inline review comments._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

Commit:
Change README.md file

zbahati reviewed (commented):
I fixed the README FILE

ouasamine commented on README.md:
```diff
@@ -1 +1,122 @@
+
+<div align="center">
+
+  <img src="images/images.png" alt="logo" width="140"  height="auto" />
+  <br/>
```
- [ ] As requested by the previous reviewer, you need to delete this img tag because it's not displaying correctly.

ouasamine commented on README.md:
```diff
@@ -1 +1,122 @@
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="https://zbahati.github.io/Hello-Microverse/">visit website</a>
+
+> Describe your project in 1 or 2 sentences.
+
```
- [ ] As requested by the previous reviewer, these descriptive lines are meant to guide you, you need to remove them after adding the README to your project.

ouasamine commented on README.md:
```diff
@@ -1 +1,122 @@
+- LinkedIn: [LinkedIn](https://linkedin.com/in/zirimwabagabo-bahati)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+
```
- [ ] The previous reviewer requested some changes to the "future features", "contributing" and "support" sections, but I noticed that you deleted them completely, these sections are mandatory and should not be deleted please add them back and customise them properly.

ouasamine commented on README.md:
```diff
@@ -1 +1,122 @@
+
+
+
+### Run tests
+
+You need to push the project to the github then the system will make the test
+First you need to create branch by using
+git checkout -b branch-name,
+then use the following command to push you branch
+
+- git add .
+- git commit -m 'add your message'
+- git push
```
- [ ] [OPTIONAL] Your project doesn't include testing, so you can delete this sub-section and keep only the "setup" and "prerequisites".

ouasamine reviewed (changes requested):
Hi @zbahati 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![](https://media.giphy.com/media/t8YPf3ACKeouLzEIvr/giphy.gif)

 ## Highlights
- Nice PR title and summary ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

zbahati committed:
Updating README.md  

Fixing Contributors section, support section, removing run test section of README file 
Remove unnecessary part of the files

shella12 commented on README.md:
```diff
@@ -1 +1,94 @@
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+> Add a link to your deployed project.
+
```
- [ ] As requested by the previous reviewer, please remove the default comment like 

>  Add a link to your deployed project

>  Describe how a new developer could make use of your project. (line 28)

from your README. They were used in the template to explain to the user the purpose of each section. You can go ahead and remove it from your README

shella12 commented on README.md:
```diff
@@ -1 +1,94 @@
+- LinkedIn: [LinkedIn](https://linkedin.com/in/zirimwabagabo-bahati)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+ [ ] **[html-makeup-feature]**
+- [ ] **[style-feature]**
+- [ ] **[Fixing-bugs-Feature]**
+
```
- [ ] [OPTIONAL] Please include the heeading of Future Feature section as it is very difficult to tell wha is inlisted without the heading. To add heading use `#` e.g.
```sh

<!-- FUTURE FEATURES -->

## Future Feature
 [ ] **[html-makeup-feature]**
- [ ] **[style-feature]**
- [ ] **[Fixing-bugs-Feature]**

```

shella12 commented on README.md:
```diff
@@ -1 +1,94 @@
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./License.md) licensed.
+
```
- [ ] The link to your `MIT LICENCE` is broken. Please use these [guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to correctly add LICENCE into your README.

shella12 commented on README.md:
```diff
@@ -1 +1,94 @@
+Your support is greatly appreciated and helps us continue to improve this project.
+Here are three ways you can support us:
+
+Star the project on GitHub to show your appreciation
+Contribute to the project by submitting bug reports or feature requests
+Spread the word by telling your friends and colleagues about our project
+Thank you for your support!
+
+
```
- [ ] Please include `Acknowledment section ` in your README to give credit where credit is due.

shella12 commented on README.md:
```diff
@@ -1 +1,94 @@
-# Hello-Microverse
\ No newline at end of file
+<a name="readme-top">SET UP LINTERS AND CREATE PULL REQUEST</a>
+
+<!-- PROJECT DESCRIPTION -->
```
- [ ] [OPTIONAL] I would strongly recommend you to include `Table of content` from the [readme template.](https://github.com/microverseinc/readme-template) The table of contents serves two purposes: It gives users an overview of the document's contents and organisation. It allows readers to go directly to a specific section of an on-line document.

```
# 📗 Table of Contents

- [📖 About the Project](#about-project)
  - [🛠 Built With](#built-with)
    - [Tech Stack](#tech-stack)
    - [Key Features](#key-features)
  - [🚀 Live Demo](#live-demo)
- [💻 Getting Started](#getting-started)
  - [Setup](#setup)
  - [Prerequisites](#prerequisites)
  - [Install](#install)
  - [Usage](#usage)
  - [Run tests](#run-tests)
  - [Deployment](#triangular_flag_on_post-deployment)
- [👥 Authors](#authors)
- [🔭 Future Features](#future-features)
- [🤝 Contributing](#contributing)
- [⭐️ Show your support](#support)
- [🙏 Acknowledgements](#acknowledgements)
- [❓ FAQ (OPTIONAL)](#faq)
- [📝 License](#license)
```

shella12 reviewed (changes requested):
Hi @zbahati , 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet. You are almost there...

<img src="https://media1.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif"/>

## Highlights

- All linters passed ✅   

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
Fixing README and Fix the Link of MIT license

Commit:
Merge branch 'test-pull-request' of https://github.com/zbahati/Hello-Microverse into test-pull-request

Commit:
Modify Read me file

mukhammadrizooff reviewed (approved):
Hi @zbahati, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<img src="https://media4.giphy.com/media/YKdm91XONLTaw/giphy-downsized-medium.gif"/>

## Highlights
- No linter errors ✔️ 
- Correct GitHub Flow ✔️ 
- Good implementations ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
